### PR TITLE
task solution

### PR DESCRIPTION
**Pull request by Gabai62**
- [DEMO LINK](https://Gabai62.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Gabai62.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,18 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input data-qa="keypress" class="search-form__input search-form__input_big"
+          type="text" placeholder="Try &quot;Los Angeles&quot;">
+      </label>
```
Comment (volodymyr-soltys97) [inline src/index.html:18]: Need to fix code style
```suggestion
    <form
      class="search-form search-form_big"
      data-qa="big"
    >
      <label class="search-form__label search-form__label_big">
        <input
          data-qa="keypress"
          class="search-form__input search-form__input_big"
          type="text" 
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,18 @@
...
+    </form>
+
+    <form class="search-form search-form_small" data-qa="small">
+      <label class="search-form__label search-form__label_small">
+        <input data-qa="hover" class="search-form__input search-form__input_small"
+          type="text" placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
```
Comment (volodymyr-soltys97) [inline src/index.html:26]: The same as above, also no need to use tag form here, you can use only one tag form for this task

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Looks good

**Pull request closed without merging**